src/forklift/cli.py

```diff
@@ -179,30 +206,30 @@ def _create_default_config(folders):
         return abspath(json_config_file.name)
 
 
-def _get_config_folders():
-    return get_config_prop('paths')
+def _get_repos():
+    return get_config_prop('repositories')
 
 
-def _validate_config_folder(folder, raises=False):
-    if exists(folder):
+def _validate_repo(repo, raises=False):
+    url = _repo_to_url(repo)
+    response = get(url)
+    if response.status_code == 200:
         message = '[Valid]'
     else:
-        message = '[Folder not found]'
+        message = '[Invalid URL]'
```

Comment:
This is confusing. I would say something like invalid username or organization and repository eg: agrc/parcels. since that is what the user is sending in its confusing to see invalid url. 

I passed https://github.com/agrc/Parcels.git and it said invalid url.